README.md

```diff
@@ -21,51 +21,80 @@ You can download this repository or clone it onto your local machine. Cloning th
   1. Open a terminal or shell in the directory where you want this repository to be in. 
   2. Run the command `git clone https://github.com/o3de/o3de.org.git`.
 
-### Setup Hugo, npm, and bootstrap
+### Setup Hugo, npm, and dependencies
 1. To install **Hugo (extended version)**, follow the instructions for your machine in the [Hugo documentation](https://gohugo.io/getting-started/installing). 
    
     *Note: You must install the **extended version** of Hugo.*
 
-1. To install **npm**, following the instructions in the [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) documentation. Installing npm also installs **nodejs**. 
+2. To install **npm**, follow the instructions in the [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) documentation. Installing npm also installs **Node.js**.
 
-2. To install **bootstrap**, run the following command in a terminal or shell in the root of your o3de.org clone: `npm install bootstrap`.
+3. To install dependencies, run the following command from the `o3de.org` repository:
+
+    ```shell
+    cd <path-to-repo>/o3de.org
+    npm install
+    ```
 
-    *Note: `npm` is only one way to install bootstrap. You can use other package managers to install bootstrap, such as [yarn](https://yarnpkg.com/package/bootstrap).*
 
 ### Build the site
 1. Open a terminal or shell and navigate to the `o3de.org` repository.
    
-```shell
-$ cd <path-to-repo>\o3de.org
-```
+    ```shell
+    cd <path-to-repo>/o3de.org
+    ```
    
 2. Run the command `hugo serve`. 
```

Comment:
It always seemed odd to me that we instructed readers to use the command `hugo serve`. The hugo command is actually `server`. While using the shorter version appears to work, we should probably use the actual command in our documentation.
```suggestion
2. Run the command `hugo server`. 
```

Reply:
Yes we should use the correct command. I think I just got into the habit of using `serve` :P